### PR TITLE
Test pull request

### PR DESCRIPTION
**Pull request by turalf**
Test pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Kanban
\ No newline at end of file
+# Kanban
+
+# Add new section in test branch
```
Comment (bhr94) [inline README.md:3]: please change the section name

Comment (turalf) [reply README.md:3]: On it. changing

**Comment by bhr94** [comment]
please fix the conflict

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # Kanban
-hellog 
\ No newline at end of file
+
```
Comment (bhr94) [inline README.md:2]: jejkjkh

**Pull request merged**